e2e/template.go

```diff
@@ -257,6 +257,7 @@ func baseTemplate(location string) *datamodel.NodeBootstrappingConfiguration {
 			PodInfraContainerImageURL: "mcr.microsoft.com/oss/kubernetes/pause:3.6",
 			HyperkubeImageURL:         "mcr.microsoft.com/oss/kubernetes/",
 			WindowsPackageURL:         "windowspackage",
+			LinuxPrivatePackageURL:    "linuxPrivatepackagesURL",
```

Comment:
shouldn't this be empty in the base template and only set during scenarios where we actually use a VHD with private pkgs installed?

Reply:
set it to empty string